### PR TITLE
Remove policy/v1beta1 since the min supported k8s version supports policy/v1

### PR DESCRIPTION
**Pull request by rdrgmnzs**
### Description

Remove policy/v1beta1 since as of version 2.5.x the min supported k8s version 1.22 supports policy/v1.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [X] Manually tested
- [X] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by johngmyers** [comment]
I don't see how this "fixes" the check, but it does make it easier to tell `helm template` that the capability exists.

/ok-to-test
/lgtm

**Comment by kishorj** [comment]
policy/v1 is available since k8s 1.21. The min version we support with the latest release v2.5.x is k8s 1.22 or later. We can remove the policy/v1beta1 completely from the helm template.

File: helm/aws-load-balancer-controller/templates/pdb.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if and .Values.podDisruptionBudget (gt (int .Values.replicaCount) 1) }}
-{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/templates/pdb.yaml:2]: We can remove this capability check completely and explicitly use policy/v1

**Comment by rdrgmnzs** [comment]
Ok, went ahead and removed the conditional so we explicitly rely on policy/v1.

**Review by kishorj — APPROVED** [review]

**Pull request merged**